tests/unit/components/dialog/GitCommitDialog.spec.js

```diff
@@ -8,14 +8,17 @@ import FileStatus from 'src/models/git/FileStatus';
 
 installQuasarPlugin();
 const mockStagedFileStatus = new FileStatus({ headStatus: 0, workdirStatus: 2, stageStatus: 2 });
-const mockNotStagedFileStatus = new FileStatus({ headStatus: 1, workdirStatus: 0, stageStatus: 1 });
+const mockNotStagedFileStatus = new FileStatus({ headStatus: 1, workdirStatus: 1, stageStatus: 1 });
+const mockDeletedFileStatus = new FileStatus({ headStatus: 0, workdirStatus: 0, stageStatus: 3 });
 
 jest.mock('src/composables/events/DialogEvent', () => ({
   subscribe: jest.fn(),
 }));
 
 jest.mock('src/composables/Project', () => ({
-  getStatus: jest.fn(() => Promise.resolve([mockStagedFileStatus, mockNotStagedFileStatus])),
+  getStatus: jest.fn(() => Promise.resolve([mockStagedFileStatus,
```

Comment:
Jump for each element of the array

Reply:
Done

Reply:
done on top, why not on bottom... don't skip legs day !